lua/packer/display.lua

```diff
@@ -101,6 +101,10 @@ local keymaps = {
   prompt_revert = {
     rhs = '<cmd>lua require"packer.display".prompt_revert()<cr>',
     action = 'revert an update'
+  },
+  retry = {
+    rhs = '<cmd>lua require"packer.display".retry()<cr>',
+    action = 'retry'
```

Comment:
Nitpick: could we be more specific with the action name, i.e. "retry failed operations"?